library/psa_crypto_se.c

```diff
@@ -75,6 +90,123 @@ const psa_drv_se_t *psa_get_se_driver( psa_key_lifetime_t lifetime )
 
 
 
+/****************************************************************/
+/* Slot management */
+/****************************************************************/
+
+/** The type of bit vector elements. A bit vector is represented as an
+ * array of bit vector elements. */
+typedef unsigned bv_element_t;
+/* Number of bits per bit vector elements. Assumes no padding bits,
+ * which Mbed TLS checks in selftest.c. */
+#define BV_BITS_PER_ELEMENT ( sizeof( bv_element_t ) * CHAR_BIT )
+
+struct psa_drv_se_slot_usage_s
+{
+    size_t size; /** Number of elements of \c vector */
+    bv_element_t vector[1]; /* Should be vector[] but we don't allow C99 */
+};
+
+static psa_key_slot_number_t su_size(
+    const psa_drv_se_slot_usage_t *slot_usage )
+{
+    return( slot_usage->size * BV_BITS_PER_ELEMENT );
+}
+
+static int su_get( const psa_drv_se_slot_usage_t *slot_usage,
+                   psa_key_slot_number_t n )
+{
+    bv_element_t elt = slot_usage->vector[n / BV_BITS_PER_ELEMENT];
+    return( elt >> (n % BV_BITS_PER_ELEMENT) & 1 );
+}
+
+static void su_set( psa_drv_se_slot_usage_t *slot_usage,
+                    psa_key_slot_number_t n,
+                    int value )
+{
+    bv_element_t *elt = &slot_usage->vector[n / BV_BITS_PER_ELEMENT];
+    bv_element_t mask = 1 << (n % BV_BITS_PER_ELEMENT);
+    if( value )
+        *elt = *elt | mask;
+    else
+        *elt = *elt & ~mask;
+}
+
+/* Callback function for drivers */
```

Comment:
For drivers to implement? for drivers to use? to copy paste? What should drivers do with this?

Reply:
Moot because this chunk of code is no longer present.